### DISCUSSION TITLE
Unable to get verified for Student Developer Pack

### DISCUSSION THREAD

**Discussion by JhaNishant** [community/community#110809] [GitHub Education]
### Hi everyone,

I am trying to verify myself for the student pack. During the upload proof process I use the continuity camera from Macbook and use the iPhone camera to take the picture of my enrollment letter which has all the details including dates, addresses and course details. Still I get an error. 

- You are unlikely to be verified until you have completed your [GitHub billing information] with your full name exactly as it appears in your academic affiliation document. If you have only a single legal name, enter it in both the first and last name fields. You do not have to add a payment method. You may need to log out and log back in to GitHub before re-applying.
- The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect email domain, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.
- You appear not to be near any campus location for the school you have selected. Your academic affiliation documentation must indicate virtual learning if you are remote. Your must configure your browser and operating system to allow location access. You may not use a VPN. If you believe that our information about your school is incomplete or incorrect e.g. missing campus location(s), then please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.

None of these issues apply to me. Hence I am stuck.
Much thanks in advance for help.

**Comment by thekarthikr** [comment]
I guess, you got your student developer pack.

**Comment by Yerock09** [comment]
Puesla verdad no y está bien si me vas ah hackear ya no puedo hacer nada

**Comment by notSaqlain** [comment]
I understand you're facing difficulties verifying your student status for the GitHub Student Developer Pack, even though you've used a clear picture with all necessary details. It appears the system might be identifying potential discrepancies based on the information you've provided. Here's what you can do:

**1. _Double-check your information:_** Ensure your full name on your GitHub account matches exactly with the name on your enrollment letter, including any special characters or middle names.

**2. _Review your school selection:_** Verify that you've selected the correct institution during the application process. If there's any discrepancy, choose the option "My selected school has incorrect or incomplete information" when submitting a support ticket.

**3. _Address location issues (if applicable):_**
-> If you're a remote student: Check if your enrollment letter specifically mentions "virtual learning." If not, try obtaining a document from your school confirming your remote learning status.
-> Disable VPN or location restrictions: If you use a VPN or have location access restricted on your device, temporarily disable them during the verification process.
4. Contact GitHub Education support: If none of the above solutions work and none of the provided error messages directly apply to your situation, contact GitHub Education support directly. Explain the issue and attach pictures of your enrollment letter and any relevant documents. You can access support here: https://github.com/topics/google-docs.

Additional tips:

Try logging out and logging back into GitHub after making any changes.
Clear your browser cache and cookies before reapplying.
Remember, patience and clarity are key when dealing with verification processes. By following these steps and providing clear explanations to support, you should be able to successfully verify your student status and access the benefits of the Student Developer Pack.

**[PLEASE MARK AS COMPLETE AND CLOSE THE DISCUSIION IF THIS HELPS!]**

**Comment by Yerock09** [comment]
Yes, everything is correct.
